build/yaml/crd/nsx.vmware.com_vpcnetworkconfigurations.yaml

```diff
@@ -0,0 +1,140 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.9.2
```

Comment:
The kubebuilder version is different with others, could we unify it?

Reply:
I plan to submit a separate patch to update the controller-gen package version in all CRD yaml files. Also update the license format.

Reply:
The latest released version is 0.9.2. I plan to update the controller-gen version to 0.9.2.